packages/base/src/values.ts

```diff
@@ -0,0 +1,74 @@
+// This module provides a ValueObject implementation for CKB related data
```

Comment:
please [mv](https://stackoverflow.com/questions/433111/how-to-make-git-mark-a-deleted-and-a-new-file-as-a-file-move) the file from `values.js` instead of delete it

Reply:
I update the commit order
1. rename `.d.ts` or `.js` to `ts`, and commit it.
2. delete unused js file and commit it
3. modify the ts source file and commit it.

Same result like origin.
![Uploading image.png…]()